recipes/gvisor/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set provider = "gvisor" %}
+{% set goname = "github.com/google/"+provider.lower() %}
+{% set version = "20200323.0" %}
+
+{% set name = goname.split('/')[-1] %}
+{% set pkg_src = ('src/'+goname).replace("/",os.sep) %}
```

`gname` and `pkg_src` were not being used anywhere else.
```suggestion
{% set version = "20200323.0" %}
```